.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
+    - os: osx # OSX 10.11
+      osx_image: xcode7.2
+      dotnet: 1.0.0-preview2-003121
+
+script:
```

Comment:
the `dotnet --info` it's already done by travis after the install the .net core sdk.
I'll remove it from script so it's clear is useless